Pod/Classes/VOKMappableModel.h

```diff
@@ -0,0 +1,32 @@
+//
+//  VOKMappableModel.h
+//  VOKCoreData
+//
+
+#import "VOKCoreDataManager.h"
+
+/**
+ *  Any models that conform to this protocol will be automatically registered for mapping with the shared instance 
```

Comment:
what's the performance of loading (lots of) the maps?  I could see a potential problem of an unexpected slowdown when first accessing the shared instance of the VOKCoreDataManager if the maps take a long time to load/create.  whereas if you knew the maps took a long time to load, you could manually trigger them to load in a background queue.  I have no idea if this is an actual performance issue or my paranoid fantasy.